core/settings/base.py

```diff
@@ -34,6 +34,10 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'apps.api',
+    'apps.account',
+    'apps.shared',
+    'apps.rating',
+    'apps.timetable',
 ]
```

Comment:
@andela-snwuguru A popular rule-of-thumb for naming apps in a django project is to name homogeneous apps (apps that handle/contain one type of content e.g _accounts_, _reviews_, _timetables_) in plural, and then name heterogeneous apps (apps that handle/contain multiple types of content e.g _api_, or _shared_).
I know